titles/admin-rhdh/title-admin.adoc

```diff
@@ -18,6 +18,9 @@ include::assemblies/assembly-install-rhdh-ocp.adoc[leveloffset=+1]
 
 include::assemblies/assembly-enabling-authentication.adoc[leveloffset=+1]
 
+// Running RHDH behind a proxy
+include::admin/proc-running-rhdh-behind-a-proxy.adoc[leveloffset=+1]
```

Comment:
Is the path fixed? I can still see the error on the preview website: `Unresolved directive in title-admin.adoc - include::admin/proc-running-rhdh-behind-a-proxy.adoc[leveloffset=+1]`
https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-225/admin-rhdh/

Reply:
The path is now fixed and you can view the procedure. Sorry, some content restructuring happened and i forgot to include the correct path.